helm-chart/renku/templates/ingress.yaml

```diff
@@ -59,53 +53,54 @@ spec:
             port:
               name: {{ $keycloakServicePort }}
       {{- end }}
-      - path: /gitlab
```

Comment:
Moved further down in the file in an if statement that templates this only when v1 services are enabled.